panic when defining a type with empty description

When specifying a type but leaving description empty, like this:
```yaml
controller:
  # -- (int)
  replicas:
```
helm-docs panics:
```
time="2021-07-30T19:02:11Z" level=info msg="Generating README Documentation for chart /helm-docs"
panic: runtime error: slice bounds out of range [6:5]

goroutine 18 [running]:
github.com/norwoodj/helm-docs/pkg/document.parseNilValueType(0xc0002fd000, 0x39, 0xc0001bd9f5, 0x5, 0x0, 0x0, 0xc0001bd9f5, 0x5, 0x0, 0x0, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:87 +0x257
github.com/norwoodj/helm-docs/pkg/document.createValueRow(0xc0002fd000, 0x39, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc0001bd9f5, 0x5, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:154 +0x4a9
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromField(0xc0002fd000, 0x39, 0xc000214aa0, 0xc000214b40, 0xc00016bd40, 0x1, 0xc0003076c0, 0x1, 0x1, 0x0, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:317 +0x914
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromObject(0xc0002dcf20, 0x17, 0xc000214460, 0xc000214500, 0xc00016bd40, 0x1, 0xc0003072d0, 0x1, 0x1, 0x0, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:282 +0x6f0
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromField(0xc0002dcf20, 0x17, 0xc000214460, 0xc000214500, 0xc00016bd40, 0x1, 0xc0003072d0, 0x1, 0x1, 0x0, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:303 +0x152
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromObject(0xc00029d310, 0xb, 0xc000211e00, 0xc000211ea0, 0xc00016bd40, 0x1, 0xc000304000, 0x3d, 0x49, 0x0, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:282 +0x6f0
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromField(0xc00029d310, 0xb, 0xc000211e00, 0xc000211ea0, 0xc00016bd40, 0x1, 0x52, 0xaa, 0x49, 0x0, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:303 +0x152
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromObject(0x0, 0x0, 0x0, 0xc0001cb720, 0xc00016bd40, 0x1, 0x69a605, 0xc0002ca800, 0x203000, 0x40d630, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:282 +0x6f0
github.com/norwoodj/helm-docs/pkg/document.createValueRowsFromField(0x0, 0x0, 0x0, 0xc0001cb720, 0xc00016bd40, 0x1, 0xc0001c52b0, 0xc0001213e8, 0x69812f, 0xc0001213f8, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/values.go:303 +0x152
github.com/norwoodj/helm-docs/pkg/document.getSortedValuesTableRows(0xc0001cb720, 0xc00016bd40, 0x6ff22eecfa3e7f8b, 0x6f0000c00016ea00, 0x6f00000000000030, 0x1, 0x6ff22eecfa3e7f8b)
        /home/runner/work/helm-docs/helm-docs/pkg/document/model.go:32 +0x65
github.com/norwoodj/helm-docs/pkg/document.getChartTemplateData(0xc0000b7f08, 0x2, 0xc0000b7f5a, 0x6, 0xc0000b7fb0, 0x9, 0xc0001bc004, 0x6, 0x0, 0xc000114d20, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/model.go:85 +0xcb
github.com/norwoodj/helm-docs/pkg/document.PrintDocumentation(0xc0000b7f08, 0x2, 0xc0000b7f5a, 0x6, 0xc0000b7fb0, 0x9, 0xc0001bc004, 0x6, 0x0, 0xc000114d20, ...)
        /home/runner/work/helm-docs/helm-docs/pkg/document/generate.go:44 +0x2bc
main.retrieveInfoAndPrintDocumentation(0x889ba5, 0x1, 0xc0000b79c0, 0xa, 0xc0000978d0, 0x1, 0x1, 0xc0000b7ed0, 0x6010100)
        /home/runner/work/helm-docs/helm-docs/cmd/helm-docs/main.go:26 +0x2d5
created by main.helmDocs
        /home/runner/work/helm-docs/helm-docs/cmd/helm-docs/main.go:69 +0x4f1
```

--- a/pkg/document/values.go
+++ b/pkg/document/values.go
@@ -84,7 +84,11 @@ func parseNilValueType(key string, description helm.ChartValueDescription, autoD
 
 	if len(t) > 0 {
 		t = t[1 : len(t)-1]
-		description.Description = description.Description[len(t)+3:]
+		if len(description.Description) > len(t)+3 {
+			description.Description = description.Description[len(t)+3:]
+		} else {
+			description.Description = ""
+		}
 	} else {
 		t = stringType
 	}

--- a/pkg/document/values_test.go
+++ b/pkg/document/values_test.go
@@ -931,18 +931,20 @@ animals:
   birds:
   birdCount:
   nonWeirdCats:
+  undescribedCount:
 	`)
 
 	descriptions := map[string]helm.ChartValueDescription{
-		"animals.birdCount":    {Description: "(int) the number of birds we have"},
-		"animals.birds":        {Description: "(list) the list of birds we have"},
-		"animals.nonWeirdCats": {Description: "the cats that we have that are not weird"},
+		"animals.birdCount":        {Description: "(int) the number of birds we have"},
+		"animals.birds":            {Description: "(list) the list of birds we have"},
+		"animals.nonWeirdCats":     {Description: "the cats that we have that are not weird"},
+		"animals.undescribedCount": {Description: "(int)"},
 	}
 
 	valuesRows, err := getSortedValuesTableRows(helmValues, descriptions)
 
 	assert.Nil(t, err)
-	assert.Len(t, valuesRows, 3)
+	assert.Len(t, valuesRows, 4)
 
 	assert.Equal(t, "animals.birdCount", valuesRows[0].Key)
 	assert.Equal(t, intType, valuesRows[0].Type)
@@ -964,6 +966,13 @@ animals:
 	assert.Equal(t, "", valuesRows[2].AutoDefault)
 	assert.Equal(t, "the cats that we have that are not weird", valuesRows[2].Description)
 	assert.Equal(t, "", valuesRows[2].AutoDescription)
+
+	assert.Equal(t, "animals.undescribedCount", valuesRows[3].Key)
+	assert.Equal(t, intType, valuesRows[3].Type)
+	assert.Equal(t, "`nil`", valuesRows[3].Default)
+	assert.Equal(t, "", valuesRows[3].AutoDefault)
+	assert.Equal(t, "", valuesRows[3].Description)
+	assert.Equal(t, "", valuesRows[3].AutoDescription)
 }
 
 func TestNilValuesWithDefaults(t *testing.T) {

